src/main/java/org/thoughtslive/jenkins/plugins/jira/Site.java

```diff
@@ -56,12 +58,13 @@
   private transient JiraService jiraService = null;
 
   @DataBoundConstructor
-  public Site(final String name, final URL url, final String loginType, final int timeout) {
+  public Site(final String name, final URL url, final String loginType, final int timeout, final int readTimeout) {
```

Comment:
Can you also help update the https://jenkinsci.github.io/jira-steps-plugin/getting-started/config/authentication/ doc please? 

The source code is at https://github.com/jenkinsci/jira-steps-plugin/blob/master/hugo/content/getting-started/config/authentication.md